src/sharedTypes.ts

```diff
@@ -28,8 +28,9 @@ export interface Transcription {
 
 export interface Word {
   word: string;
-  startTime: number;
+  inputStartTime: number;
```

Comment:
Have you confirmed changing this won't break anything with export?

Reply:
We will fix the export after this PR is merged